abb_hardware_interface/src/abb_hardware_interface.cpp

```diff
@@ -216,10 +216,9 @@ std::vector<hardware_interface::StateInterface> ABBSystemHardware::export_state_
     {
       for (auto& joint : unit.joints)
       {
-        // TODO(seng): Consider changing joint names in robot description to match what comes
```

Comment:
How will the `seng` now know what to fix :stuck_out_tongue:?
Joke aside, I personally wouldn't remove the original code but rather only comment it out and put our comment beside why we changed it. At least from my experience this helps when it's the need to sync the code with the original repo in the future and you need to resolve the conflicts.

Reply:
Meh, I would leave it give that this is a (public) fork of a different repo. All the original code is there anyway

Reply:
The diff can always be seen, so I don't think it's necessary to comment out. Just removing it is fine IMO.

However, with these things, only rarely something happens upstream without oneself taking action. When modifying original code, I would suggest to at least post an issue, or better a PR (gets more attention and comes with diff) in the original repo. The "worst" that can happen is rejection, but then it comes with an answer to why (actually, worse would be if it gets totally ignored, but with this repo I think the chances are good to get feedback at least). In the ideal case, it gets fixed upstream.